api/src/test/java/org/openmrs/api/db/hibernate/HibernateFormDAOTest.java

```diff
@@ -43,4 +45,18 @@ public void shouldFilterAgainstFormFields() {
 		assertEquals(0, (Object)dao.getForms(null, false, Collections.emptyList(), null, formFields, formFields, Arrays.asList(new Field(3))).size());
 		
 	}
+
+	@Test
+	public void shouldGetFormFieldsByForm() {
+		Form form = new Form(2); 
+		List<FormField> formFields = dao.getFormFields(form);
+
+		assertNotNull(formFields);
+
+		final int expectedSize = 2;
```

Comment:
And our naming convention for constants is using capital letters. 😊

Reply:
done